hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/scm/TestStorageContainerManager.java

```diff
@@ -733,7 +727,7 @@ public void testSCMReinitializationWithHAEnabled() throws Exception {
       // validate there is no ratis group pre existing
       try {
         validateRatisGroupExists(conf, clusterId);
```

Comment:
Thanks @hemantk-12 for the review.  I agree that this can be changed to `assertThrows`, but would like to limit the scope of change here.

Reply:
Filed HDDS-10090 for possible cleanup in `TestStorageContainerManager`.  Please feel free to add items.  I'll unassign it from myself when this is merged, so that new contributors (or anyone else) can take this up.